ast/dune

```diff
@@ -6,12 +6,9 @@
  (name        ppxlib_ast)
  (public_name ppxlib.ast)
  (libraries
-  ocaml-compiler-libs.shadow
-  ocaml-compiler-libs.common
-  compiler-libs.common
-  ocaml-migrate-parsetree
+  ppxlib_astlib
   stdlib-shims)
- (flags (:standard -open Ocaml_shadow -safe-string) -w -9-27-32)
```

Comment:
This open was for the developers of ppxlib, so we should keep it

Reply:
Ok, thanks! I've put it back.